pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -183,6 +196,28 @@ func (jsonVM *jsonVM) toVM(project string, opts *ProviderOpts) (ret *vm.VM) {
 		Project:     project,
 		SQLPort:     config.DefaultSQLPort,
 		AdminUIPort: config.DefaultAdminUIPort,
+=======
+		Name:                   jsonVM.Name,
+		CreatedAt:              jsonVM.CreationTimestamp,
+		Errors:                 vmErrors,
+		DNS:                    fmt.Sprintf("%s.%s.%s", jsonVM.Name, zone, project),
+		Lifetime:               lifetime,
+		Preemptible:            jsonVM.Scheduling.Preemptible,
+		Labels:                 jsonVM.Labels,
+		PrivateIP:              privateIP,
+		Provider:               ProviderName,
+		ProviderID:             jsonVM.Name,
+		PublicIP:               publicIP,
+		RemoteUser:             remoteUser,
+		VPC:                    vpc,
+		MachineType:            machineType,
+		Zone:                   zone,
+		Project:                project,
+		SQLPort:                config.DefaultSQLPort,
+		AdminUIPort:            config.DefaultAdminUIPort,
+		NonBootAttachedVolumes: volumes,
+		LocalDisks:             localDisks,
+>>>>>>> 89b3be749c0 (roachprod: add `--gce-use-spot`)
```

Comment:
👀

Reply:
I blame github outage; it pushed it partially. Fixed now.